html-relative-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Contact Us</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="/html-relative-links/index.html">Home Page</a></li>
+    <li><a href="/html-relative-links/about-me/about-me.html">About Me</a></li>
+    <li><a href="/html-relative-links/projects/projects.html">Projects</a></li>
```

Comment:
you can traverse through directories using `../` to get to a parent directory instead of `/html-relative-links/`  (`../../` will go up two directories to the grandparent).

Let's use the `../` notation to traverse relative links in all your html files.